src/extension/module.js

```diff
@@ -1725,6 +1725,19 @@
     }
   }
 
+  /**
+   * Event listner for postMessage events.
+   * @param {Object} options The options
+   */
+
+  async function handlePostMessage(options) {
+    chrome.runtime.onMessage.addListener(async (message) => {
```

Comment:
Note that we are introducing an extension-only feature here, but `module.js` will also still be loaded as a simple bookmarklet. Therefore, any `chrome` object access needs to be safeguarded.